addon/styles/components/text-field.scss

```diff
@@ -1,76 +1,122 @@
 .cs-component-text-field {
```

Comment:
For these changes, I pretty much killed the original scss for TextField, and replaced it with the Email scss (implemented by @burieberry) but renamed anything email specific to text-field